src/simulator/base.py

```diff
@@ -0,0 +1,32 @@
+import object
+from position import Position
+from shape import Shape
+import object_registry
+from robot import Robot
+import numpy as np
+
+RadioID = str  # type aliasing, since we might change RadioID composition
+
+
+class Base(object.Object):
+    """
+    Object for keeping the robot and moving between it, etc.
+    The robot receives information from this object, such as distance.
```

Comment:
@mehdi138r 
The first sentence is not clear to me. Perhaps something along the lines of:
A physical station, primarily in charge of communicating with robots and facilitating their navigation by transmitting their (estimated) distances with the base.

Reply:
the sentences you wrote are excellent, I will use them